app/routes/tasks/edit-task.js

```diff
@@ -1,12 +1,18 @@
-// app/routes/tasks/edit-task.js
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 
 export default class TasksEditTaskRoute extends Route {
   @service store;
 
+  // need to reset the set options for priority and status within the controller
+  // so the modal displays the correct default selections
+  setupController(controller, model) {
+    super.setupController(controller, model);
+    controller.resetDefaults();
```

Comment:
Ember has a special hook for resetting the state: 
https://api.emberjs.com/ember/5.12/classes/Route/methods/resetController?anchor=resetController

Reply:
Done.
Made this change to create-task and edit-task where I need my state and priority options reset

RE: https://github.com/Mr-Smyth/task-manager/pull/25/commits/7b0335f022b385f802f0aa58205a8f50803cbff5